# Generated by Django 4.0.10 on 2023-12-04 02:26

from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        ("taskmonitor", "0007_alter_tasklog_exception"),
    ]

    operations = [
        migrations.CreateModel(
            name="TaskStatistic",
            fields=[
                (
                    "id",
                    models.BigAutoField(
                        auto_created=True,
                        primary_key=True,
                        serialize=False,
                        verbose_name="ID",
                    ),
                ),
                ("name", models.CharField(max_length=255)),
                ("app", models.CharField(max_length=255)),
                ("runtime_min", models.FloatField()),
                ("runtime_avg", models.FloatField()),
                ("runtime_max", models.FloatField()),
                ("runtime_total", models.FloatField()),
                ("runs_succeeded", models.IntegerField()),
                ("runs_failed", models.IntegerField()),
                ("runs_retried", models.IntegerField()),
                ("runs_total", models.IntegerField()),
            ],
            options={
                "managed": False,
            },
        ),
    ]
